include formik on todo  app

bijayqubits opened the pull request:
1.include formik and yup on todo app
2.create a sigin form
3.added some lines on readme.md file

bijayqubits committed:
include formik
1.include formik and yup on todo app
2.create a sigin form

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+    .min(8, "password must be 8 digit")
+    .required("Required"),
+});
+const onSubmit = (values) => {
+  console.log("form data:", values);
+};
+const initialValues = {
+  email: "",
+  password: "",
+};
```
It's better practice to keep ```onSubmit``` & ```initialValues``` inside component because sometimes we need component props values in side both ```onSubmit``` & ```initialValuse```

bijayqubits replied:
okay

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+        validationSchema={validationSchema}
+      >
+        {(formik) => {
+          console.log(formik);
```
It's better to remove ```console.log``` before pushing code to remote repo or put a comment why do other developer need a ```console.log``` there

bijayqubits replied:
done

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+  margin: 40px auto;
+  border-radius: 10px;
+`;
+const Head = styled.h1`
```
put one line gap

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+  padding-bottom: 3rem;
+  padding-top: 3rem;
+`;
+const Label = styled.label`
```
put one line gap

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+  margin-left: 3rem;
+  font-weight: 500;
+`;
+const Input = styled.input`
```
put one line gap

suhailqubits commented on src/components/SignInForm.js:
```diff
@@ -0,0 +1,116 @@
+  text-align: center;
+  margin: 1rem 3rem;
+`;
+const Button = styled.button`
```
put one line gap

suhailqubits reviewed (approved):
Apart from comments i made everything seems fine.

suhailqubits commented:
Fix above comments then i will merge the pull request

suhailqubits commented:
Also Please resolve conflicts

bijayqubits committed:
fixed bug and error
1. put one line space to seprate styled
2. keeped onsubmit and initialvalues inside a component
3. removed consol.log part
4. solve conflicts

bijayqubits committed:
Merge branch 'develop' into signin-form

suhailqubits committed:
Merge branch 'develop' into signin-form
